server/data/assessForEarlyReleaseApiClient.ts

```diff
@@ -324,10 +398,80 @@ export default class AssessForEarlyReleaseApiClient {
     })
   }
 
-  async getForm(prisonNumber: string, documentSubjectType: DocumentSubjectType): Promise<Buffer> {
-    return this.restClient.get<Buffer>({
-      path: `/offender/${prisonNumber}/document/${documentSubjectType}`,
-      responseType: 'arraybuffer',
-    })
+  async getForm(
+    token: string,
+    agent: Agent,
+    prisonNumber: string,
+    documentSubjectType: DocumentSubjectType,
+  ): Promise<Buffer> {
+    return this.getWithToken<Buffer>(
+      `/offender/${prisonNumber}/document/${documentSubjectType}`,
+      token,
+      agent,
+      'arraybuffer',
+    )
+  }
+
+  private getDefaultHeaders(agent: Agent) {
+    return {
+      username: agent?.username,
+      fullName: agent?.fullName,
+      role: agent?.role,
+      onBehalfOf: agent?.onBehalfOf,
+    }
+  }
+
+  private getWithToken<T>(path: string, token: string, agent: Agent, responseType = ''): Promise<T> {
```

Comment:
we add details of the user to each request as custom headers so added helper methods for each